docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,171 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives an input stream of POJOs and inserts them as rows in a database table in a fault-tolerant way.
+
+## Overview
+The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of each window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by `batchSize` property. The tuples in a window are stored in a check-pointed collection which is cleared in the `endWindow()`. The operator writes a tuple exactly once in the database.
+This operator internally extends the functionality of `AbstractPassThruTransactionableStoreOutputOperator`. AbstractPassThruTransactionableStoreOutputOperator is the base implementation for a pass-through output adapter of a transactional store. AbstractPassThruTransactionableStoreOutputOperator implementation provides the "exactly once" guarantee. "Pass-through" means it does not wait for end window to write to the store. It will begin transaction at `beginWindow` and write to the store as the tuples come and commit the transaction at `endWindow`.
```

Comment:
begin transaction => begin the transaction

Naming the base class multiple times, especially such a long name, seems needlessly verbose. How about replacing the first 2 sentences with:

The base class for this operator is `AbstractPassThruTransactionableStoreOutputOperator` which implements a pass-through output adapter for a transactional store; it guarantees exactly-once semantics.